src/components/progress-circle/progress_circle.spec.ts

```diff
@@ -0,0 +1,91 @@
+import {beforeEach, describe, expect, inject, it, TestComponentBuilder} from 'angular2/testing';
+import {Component, DebugElement} from 'angular2/core';
+import {MdProgressCircle} from './progress_circle';
+
+
+export function main() {
+  describe('MdProgressCircular', () => {
+    let builder:TestComponentBuilder;
+
+    beforeEach(inject([TestComponentBuilder], (tcb:TestComponentBuilder) => {
+      builder = tcb;
+    }));
+
+    it('applies a mode of "determinate" if no mode is provided.', (done:() => void) => {
```

Comment:
Done

I was just confident that it was going to do these things, should sounds like I'm guessing :)